### PR TITLE
Adding a Value and Location displaying toolbar

### PR DESCRIPTION
**Pull request by percurnicus**
@godber 
The x/y/value display works exactly like merview: 
1) when you click the mouse the values update
2) When you click, hold, and move the values update with the mouse
3) The values do not update when the mouse is just moving around without being clicked.

I can make it so the values update with the mouse whenever it moves if you want.

### CODE REVIEW

**Comment by percurnicus** [comment]
Here are a couple of examples of what it looks like right now:
screen shot 2015-08-06 at 12 26 33 pm

screen shot 2015-08-06 at 12 25 46 pm

**Comment by godber** [comment]
What does it do for color image?

**Comment by godber** [comment]
Shows 0 for three band images.

screen shot 2015-08-06 at 8 39 06 pm

Make it should show the R, G, and B values.

**Comment by godber** [comment]
Also, command+drag no longer permits panning the image.  This could probably peacefully coexist with the click for value and click and drag for value functionality.

**Comment by percurnicus** [comment]
Can we add this image to the core set of planetary_test_data so we can test functionality with a 3 band colored image?

**Comment by percurnicus** [comment]
I have to reset the contrast/rotation/etc. after switching images

**Comment by godber** [comment]
Yes, ALL of the display parameters should be stored with the image.  They are NOT independant global parameters but attributes of displaying a specific image.  :)

**Comment by godber** [comment]
Compare this with merview zoomed in closely.  I find two issues on Linux, see if you can reproduce them on Mac.
- Zooming in really close (like 50x), moving the cursor around INSIDE a single pixel, the value changes.
- switching X11 desktops results in the x,y,value info ceasing to update.

File: pdsview/pdsview.py [hunk 1]
```diff
@@ -209,9 +249,39 @@ def __init__(self, image_set):
...
+            self.x_value.setText('X: %.0f' % (x))
+            self.y_value.setText('Y: %.0f' % (y))
+            if self.image_set.current_image.ndim == 3:
+                # Show different band values for 3 band images
+                R = str(round(value[0], 3))
+                G = str(round(value[1], 3))
+                B = str(round(value[2], 3))
+                self.pixel_value.setText('R: %s G: %s B: %s' % (R, G, B))
+            elif self.image_set.current_image.ndim == 2:
+                # Show single pixel value for 2 band images
```
Comment (bvnayak) [inline pdsview/pdsview.py:269]: `Change in Comments`:
Please update `2 band` to `single band`. (Because ndim =2 is `2 dimensional`).

**Pull request merged**